h2o-bindings/bin/custom/python/gen_stackedensemble.py

```diff
@@ -128,11 +129,18 @@ def extend_parms(parms):
 overrides = dict(
     base_models=dict(
         setter="""
-if is_type(base_models, {ptype}):
-    {pname} = [b.model_id for b in {pname}]
+def _get_id(something):
+    if hasattr(something, "key"):
+        return something.key
+    return something
+
+if not is_type(base_models, list):
+    base_models = [base_models]
+if is_type(base_models, [H2OEstimator, H2OGridSearch, str]):
```

Comment:
having `str` there confuses me.
Is it now simpler to have:
```
if is_type(base_models, [Keyed]):
    {pname} = [b.key for b in {pname}]
    self._parms["{sname}"] = {pname}
else:
    assert_is_type({pname}, [None, str])
    self._parms["{sname}"] = {pname}
```

Reply:
I was thinking about that but it wouldn't let the user to mix the `model_ids` and actual model objects in the `base_models`. Should I drop the use case with mixed `model_ids` and model objects?

Reply:
Oh, I see, good point; you can keep the mixed use case.
Just remove the assignment for `None` then:
`self._parms["{sname}"] = {pname}`
It's unnecessary now. Thx.